packages/nx-playwright/src/executors/playwright-executor/schema.d.ts

```diff
@@ -7,6 +7,7 @@ export interface PlaywrightExecutorSchema {
   headed?: boolean;
   reporter?: string;
   browser?: 'chromium' | 'firefox' | 'webkit' | 'all';
+  runner?: 'yarn' | 'npx' | 'pnpm';
```

Comment:
why would the executor care about the runner?  this file contains flags for the executor, for when the npm tests are run.